src/main/webapp/addRestaurant.html

```diff
@@ -0,0 +1,59 @@
+<!--
+Copyright 2019 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Add Restaurant</title> 
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
Let's use our CSS file instead, not theirs.

Reply:
Yes no problem! I'm remaking (since my laptop broke) our main.css so it can be linked to all our pages